src/css/typeface-inter.css

```
@@ -1,39 +1,76 @@
+/* =============================================================================
+   INTER TYPEFACE
+   Primary body font. Includes weights from Light (300) to Bold (700).
+   ============================================================================= */
+
+/* Light - 300 */
 @font-face {
-  font-family: "Inter Light";
+  font-family: "Inter";
```

Stylelint (17.11.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)